packages/backend/src/controllers/authentication.ts

```diff
@@ -161,10 +179,9 @@ export const isAuthenticated: RequestHandler = (req, res, next) => {
         next(new AuthorizationError(`Failed to authorize user`));
     }
 };
-export const unauthorisedInDemo: RequestHandler = (req, res, next) => {
-    if (lightdashConfig.mode === LightdashMode.DEMO) {
-        throw new AuthorizationError('Action not available in demo');
-    } else {
+export const allowApiKeyAuthentication: RequestHandler = (req, res, next) => {
```

Comment:
So we want to only allow the api key to be used in certain endpoints ? makes sense.